src/components/Controls/Dropdown/Dropdown.js

```diff
@@ -63,22 +63,31 @@ class Dropdown extends React.Component {
 	handleKeyDown = e => {
 		if (
 			keyCodes.tab === e.which ||
-			(/button/i.test(e.target.tagName) && e.which === keyCodes.space) ||
-			/input|textarea/i.test(e.target.tagName)
+			(/button/i.test(e.target.tagName) && [keyCodes.space, keyCodes.enter].includes(e.which)) ||
+			/input|textarea|select/i.test(e.target.tagName)
```

Comment:
Mouse click on <DropdownItem tag="select"> will close dropdown menu, so user won't be able to select anything.

Reply:
Ok, i see. Possible solution is <DropdownItem tag="select" toggle={false}> on the menuItem. Do you have another solution?

Reply:
Will it work within a keyboard? Anyway I think it's not good to expose internal logic through props. It should be handled inside.

Reply:
Yes, it'll still work. Though, I can see if I can handle this aswell.

Reply:
pushed a suggestion change in DropdownItem.js

Reply:
That's the right fix) One more thing: click/Enter on item in nested menu should close the menu (as it works on main menu).

Reply:
I've added a prop on dropdown menu that allows parentToggle to be passed.